feedback/templates/feedback_tags/import_tags.html

```diff
@@ -0,0 +1,56 @@
+{% extends "manage/base.html" %} {% load i18n %} {% block content %}
+<h1>{% translate "Import feedback tags from another course" %}</h1>
+<form method="post" action="" class="import-form">
+	{% csrf_token %} {{ form.as_p }}
+	<div id="selected-tags"></div>
+	<button type="submit" class="btn btn-success">{% trans 'Import' %}</button>
+</form>
+<div id="tag-import-preview-container">
+	{% for course_id, tags in course_tags %}
+		<div id="tag-import-preview-{{ course_id }}" class="hidden">
+			{% if tags|length > 0 %}
+				<h3>
+					{% trans 'Tags to be imported' %}
+				</h3>
+				<table class="table table-condensed table-striped">
+					<thead>
+						<tr>
+							<th>{% trans "Feedback tag" %}</th>
+							<th>{% trans "Slug" %}</th>
+							<th>{% trans "Description" %}</th>
+							<th>{% trans "Can be imported" %}</th>
+						</tr>
+					</thead>
+					<tbody>
+						{% for tag in tags %}
+							<tr>
+								<td>
+									<span
```

Comment:
It might be better to here use the colortag templatetag ( e.g. {{ tag|colortag:'no_tooltip' }} ). It would also deal with the styling (because for example now the font color doesn't change depending on the background color).

So this span could be replaced with:
```
{% if tag.canImport %}
	{{ tag|colortag:'no_tooltip' }}
{% else %}
	{{ tag|colortag:'no_tooltip,class=not-importable' }}
{% endif %}
```